app/mailers/admin_mailer.rb

```diff
@@ -1,6 +1,6 @@
 class AdminMailer < ActionMailer::Base
   add_template_helper(ApplicationHelper)
-  default from: 'no-reply@reservations.app'
+  default from: "no-reply@#{ ActionMailer::Base.default_url_options[:host] }"
```

Comment:
It would, but frankly that's acceptable to me; it's a no-reply address so having a subdomain really doesn't bother me :stuck_out_tongue:.